packages/react-aria-components/src/ListBox.tsx

```diff
@@ -183,30 +300,78 @@ function Option<T>({item}: OptionProps<T>) {
       isHovered,
       isFocusVisible: focusVisible,
       selectionMode: state.selectionManager.selectionMode,
-      selectionBehavior: state.selectionManager.selectionBehavior
+      selectionBehavior: state.selectionManager.selectionBehavior,
+      allowsDragging: !!dragState,
+      isDragging,
+      isDropTarget: droppableItem?.isDropTarget
+    }
+  });
+
+  let renderDropIndicator = dragAndDropHooks?.renderDropIndicator || (target => <DropIndicator target={target} />);
+
+  return (
+    <>
+      {dragAndDropHooks?.useDropIndicator &&
+        renderDropIndicator({type: 'item', key: item.key, dropPosition: 'before'})
+      }
+      <div
```

Comment:
Is there supposed to be an "on" drop indicator here as well? Also don't think I'm seeing a root drop indicator either

Reply:
Root drop indicator and on drop indicators aren't necessary for listbox. The option element itself is the drop indicator (and gets props for this from `droppableItem?.dropProps`). For the root, the listbox is a target via `droppableCollection?.collectionProps`. This works for listbox on mobile screen readers but not grid, which is why we use buttons there.

Reply:
Ahhhhh, right, I was initially concerned about a possible lack of announcement but it gets a description from useVirtualDrop.

Confirmed the announcements via Talkback